test/e2e/pkg/manifest.go

```diff
@@ -172,8 +172,11 @@ func LoadManifest(file string) (Manifest, error) {
 // SortManifests orders (in-place) a list of manifests such that the
 // manifests will be ordered in terms of complexity (or expected
 // runtime). Complexity is determined first by the number of nodes,
-// and then by the total number of perturbations in the network
-func SortManifests(manifests []Manifest) {
+// and then by the total number of perturbations in the network.
+//
+// If reverse is true, then the manifests are ordered with the most
+// complex networks before the less complex networks.
+func SortManifests(manifests []Manifest, reverse bool) {
```

Comment:
The naked Boolean is slightly sus in an exported function. Consider maybe:

```go
func SortManifests(manifests []Manifest, order Order) {
  // …
}

type Order bool

const (
  Ascending Order = false
  Descending Order = true
)
// …
SortManifests(manifests, Descending)
```

Same effect, but self-documenting at the call site. (And yes, I realize that right now the one call site has the name of the flag)

Reply:
yeah, this is better, but I think I'm going to leave it for now, because:
- this is super internal,
- the real solution, from my perspective, would to be implement `sort.Interface` for `[]Manifest` and use `sort.Reverse`
- I don't want to have to convert the CLI bool flag to asc/desc, which I know I'm very likely to screw up.